<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoanCalc</class>
 <widget class="QMainWindow" name="LoanCalc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>588</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>498</width>
    <height>588</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>498</width>
    <height>588</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_closeLoan">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>481</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(180, 180, 180);
  color: rgb(40, 40, 40);
  border: 2px solid rgb(180, 180, 180);
  border-radius: 15px;
  border-color: rgb(100, 100, 100);
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_totalAmount">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>341</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	color: black;
	background-color: rgb(230, 230, 230);
	border-radius: 30px;                     /* &lt;----  20px  */ 
	border: 2px solid rgb(180, 180, 180);
}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_loanTerm">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>341</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	color: black;
	background-color: rgb(230, 230, 230);
	border-radius: 30px;                     /* &lt;----  20px  */ 
	border: 2px solid rgb(180, 180, 180);
}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ineterstRate">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>341</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	color: black;
	background-color: rgb(230, 230, 230);
	border-radius: 30px;                     /* &lt;----  20px  */ 
	border: 2px solid rgb(180, 180, 180);
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_totalAmount">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>5</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Total amount</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_interestRate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Interest Rate</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_loanTerm">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Loan Term</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_calculate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>481</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(255, 151, 57);
  color: white;
  border: 2px solid rgb(180, 180, 180);
  border-radius: 15px;
  border-color: rgb(180, 180, 180);
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_totalPayment">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>460</y>
      <width>341</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	color: black;
	background-color: rgb(230, 230, 230);
	border-radius: 30px;                     /* &lt;----  20px  */ 
	border: 2px solid rgb(180, 180, 180);
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_totalPayment">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Total payment</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_overpayment">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>390</y>
      <width>341</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	color: black;
	background-color: rgb(230, 230, 230);
	border-radius: 30px;                     /* &lt;----  20px  */ 
	border: 2px solid rgb(180, 180, 180);
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_overpayment">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Overpayment</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_PerMonth">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>315</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Payment per m.</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_paymentPerMonth">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>320</y>
      <width>341</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	color: black;
	background-color: rgb(230, 230, 230);
	border-radius: 30px;                     /* &lt;----  20px  */ 
	border: 2px solid rgb(180, 180, 180);
}</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_annuity">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Annuity payment</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_differential">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>230</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Differential payment</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_totalAmount_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>(in rubles)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_totalAmount_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>(months)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_totalAmount_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>(in %)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_totalAmount_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>(in rubles)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_totalAmount_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>(in rubles)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_totalAmount_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>(in rubles)</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
